packages/dota-lua-types/types/enums-normalized.generated.d.ts

```diff
@@ -19,7 +19,10 @@ declare const SPAWN_GROUP_HANDLE_INVALID: 0;
  */
 type ABILITY_TYPES = AbilityTypes;
 
-declare const enum AbilityTypes {
+/**
+ * @compileMembersOnly
```

Comment:
This will not work for the normalized enums, it will convert `AbilityTypes.BASIC` to `BASIC` instead of `ABILITY_TYPES_BASIC` resulting in incorrect code.

Reply:
seems this will require changes in transformer, adding @compileMembersOnly will return at
```Typescript
if (!ts.isPropertyAccessExpression(node) && !ts.isElementAccessExpression(node))
        return;
```
fix it later